deploy/supervisor_docker.conf

```diff
@@ -11,8 +11,8 @@ stderr_logfile_maxbytes=0
 startsecs=5
 startretries=2
 
-[program:npm]
-command=node server.js
+[program:frontend]
+command=/code/frontend/node_modules/.bin/webpack --config webpack.config.js --watch
```

Comment:
It seems like we should be able to just run `webpack` without finding it in `node_modules`, but I can't get it to work. Weird.

Reply:
Looks like you can create a Node script and use that here. So something like `npm run watch` and then add this to `package.json`:
```
  "scripts": {
    "watch": "webpack --config webpack.config.js --watch"
  },
```

Reply:
I tried a few things to get this to work, I can add this to the package and see if it runs. It might have been how I was installing node.

Reply:
If I make this switch it now says

```
> canvas-app-explorer@0.0.0 watch
> webpack --config webpack.config.js --watch

/usr/bin/env: ‘node’: Permission denied

```
I can run node and npm fine in the container.

Reply:
Hm, okay, I'll try it.

Reply:
Well things are getting a little further. I'm getting permission errors on frontend/bundles now.

I think running it via this `npm run watch` is changing the user id from root and ends up causing permission problems if a local file system is bound as a different user id. Maybe I can move bundles one level down too out of this directory or something.